data-access/ImageRepository.cs

```diff
@@ -30,8 +30,8 @@ public async Task<IEnumerable<Image>> GetImagesAsync(uint? userId, uint editionI
 					sql,
 					new
 					{
-						UserId = userId ?? 0, // @UserId is not expanded if userId is null
-						EditionId = editionId,
+						UserId = editionUser.userId ?? 0, // @UserId is not expanded if userId is null
```

Comment:
But UserId is never null, it can be 0, but not null. Perhaps it should be nullable? I'm not sure how the code that actually performs the query will look like.

Reply:
Let me check this, because I don't really care for how it looks.  UserId is null when an unauthenticated (anonymous) user is making read requests to the API (it is currently a uint? type).  I will check the queries to see if we can simply pass the null straight through and still get the intended results.

Reply:
You may just need to change the comment, saying it's not expanded when it's 0 and that's it.

Reply:
Looks like I was able to remove all of these, and to pass the uint? directly to the query.  And I found another spot where I was unnecessarily using `?? 1` because of a poorly written query.  That is now fixed too.  Thanks!